### ISSUE TITLE
ScrollablePane, DetailsList and Sticky Header: blank space inserted and sticky obscures 1st item

### ISSUE THREAD

**Issue by aappddeevv** [OfficeDev/office-ui-fabric-react#4234]
v5.62.

If you modify the ScrollablePane DetailsList Example to remove all non DetailsList sticky headers, you find that there is inserted extra space into the ScrollPane that throws scrolling off. There is also an issue where the DetailsList sticky details list header (per the example) is sitting on top of the first DetailsList item obscuring it, but this actually only happens under some conditions I cannot determine yet. I'm not sure these two issues are related.
```typescript
 return (
      <ScrollablePane>
{ false &&        <Sticky>{ selectionDetails }</Sticky> }
{ false &&
        <TextField
          label='Filter by name:'
          // tslint:disable-next-line:jsx-no-lambda
          onChanged={ text => this.setState({ items: text ? _items.filter(i => i.name.toLowerCase().indexOf(text) > -1) : _items }) }
        />
}
{ false &&
        <Sticky>
          <h1 style={ { margin: '0px' } }>Item List</h1>
        </Sticky>
}
        <MarqueeSelection selection={ this._selection }>
          <DetailsList
...
```
Removing these "test" stickies" resulted in a lingering div:
![scrollablelist extra space](https://user-images.githubusercontent.com/1183337/37247300-36716b16-2487-11e8-9dc7-9ba870e276d4.png)
![scrollable pane html](https://user-images.githubusercontent.com/1183337/37247301-368010da-2487-11e8-875c-1df448aa4bc6.png)

**Comment by obiwan007** [comment]
The problem with the first item obscuring was fixed in my case by a padding: 1px to the container div. Strange.
Another issue was that I had to add position: relative too - otherwise the sticky was hovering over other elements above the actual scrollable.

**Comment by aappddeevv** [comment]
I'll try the 1px pad. I did add `position: relative` as modeled after the example. Without the `position: relative` in the right place, my stickies jump to the top of the frame/window on scroll. It looks like the sticky uses position so it needs a positioned parent to anchor to the right place.

**Comment by aappddeevv** [comment]
I'm hoping PR #4602 fixes this.

**Comment by michaelangotti** [comment]
@aappddeevv Can you confirm if PR #4602 has fixed this? IF so, we will close.

**Comment by leddie24** [comment]
I think this should be resolved from #4491 .  Please reopen if this is not the case.  Linking a video below with the changes from 4491 (and removing the non DetailsList stickies)

[video](https://streamable.com/sdnru)
